app/views/datasets/licences/_licence_options.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="form-group">
+  <fieldset>
+    <legend class="visually-hidden">Choose a licence for this dataset</legend>
+
+    <%= dataset_field f, @dataset,
```

Comment:
Perhaps we should set the local variable to `form` not `f` as `f` is ambiguous now we don't have the `<%= form_for @dataset, url: update_dataset_licence_path(@dataset.uuid, @dataset.name) do |f| %>` context.